### ISSUE TITLE
LinkExchange widget: `Type error: null has no properties`

### ISSUE THREAD

**Issue by robyedlin** [Userfeeds/Apps#177]
Seems related to https://github.com/Userfeeds/Apps/issues/176
```
TypeError: Cannot set property 'opener' of undefined
  at _openTargetUrl(/npm/@linkexchange/widgets@stable:48:95361)
  at Object.P(/npm/@linkexchange/widgets@stable:40:6418)
  at Object.O.invokeGuardedCallback(/npm/@linkexchange/widgets@stable:40:5697)
  at Object.O.invokeGuardedCallbackAndCatchFirstError(/npm/@linkexchange/widgets@stable:40:5812)
  at K(/npm/@linkexchange/widgets@stable:40:7465)
  at ee(/npm/@linkexchange/widgets@stable:40:7945)
  at ne(/npm/@linkexchange/widgets@stable:40:8105)
  at Array.forEach(/)
  at J(/npm/@linkexchange/widgets@stable:40:7734)
  at le(/npm/@linkexchange/widgets@stable:40:8839)
  at qt(/npm/@linkexchange/widgets@stable:40:32225)
  at Wt(/npm/@linkexchange/widgets@stable:40:22868)
  at Oo.batchedUpdates(/npm/@linkexchange/widgets@stable:40:66195)
  at ut(/npm/@linkexchange/widgets@stable:40:17010)
  at $t(/npm/@linkexchange/widgets@stable:40:23337)
  at HTMLDocument.r(./node_modules/raven-js/src/raven.js:347:1)
```
<img width="324" alt="screen shot 2018-02-08 at 11 07 12 pm" src="https://user-images.githubusercontent.com/10917203/36012614-d32dfe4e-0d24-11e8-85fb-94aeff7fbf53.png">

**Comment by jakubsta** [comment]
@robyedlin 
It should be already resolved in current stable version `0.0.168`. There are two places where we open link target url: [1](https://github.com/Userfeeds/Apps/blob/master/widgets/linkexchange-link/src/Banner/index.tsx#L167), [2](https://github.com/Userfeeds/Apps/blob/master/packages/components/src/Link.tsx#L21) and neither can throw mentioned exception.

**Comment by robyedlin** [comment]
👍
